external-crates/move/crates/move-package-alt/src/package/package_impl.rs

```diff
@@ -306,42 +309,14 @@ impl<F: MoveFlavor> Package<F> {
         // If no system dependencies are specified, we include the default system dependencies.
         Ok(F::default_system_dependencies(env.id().to_string()))
     }
+}
 
-    /// Validate the manifest contents, after deserialization.
```

Comment:
This validation should be happening for the root package and in all environments (and I think it is); this is the wrong place for it.